Package.swift

```diff
@@ -10,6 +10,12 @@
 
 import PackageDescription
 
+#if swift(<6)
+let swiftConcurrency: SwiftSetting = .enableExperimentalFeature("StrictConcurrency")
+#else
+let swiftConcurrency: SwiftSetting = .enableUpcomingFeature("StrictConcurrency")
+#endif
+
```

Comment:
No need to do that if you just specify `// swift-tools-version:6.0` at the top of the file.
See https://github.com/StanfordSpezi/Spezi/blob/main/Package.swift